src/ripple/app/ledger/Ledger.cpp

```diff
@@ -1053,7 +1048,8 @@ static void visitHelper (
     std::function<void (std::shared_ptr<SLE> const&)>& function,
```

Comment:
It's off topic for this commit, but I don't mind this minor renaming.  I've gone with `callback` both here and in `visitStateItems`.